utils/supabase.ts

```diff
@@ -190,7 +190,7 @@ export async function resetPassword(email: string) {
     // OAuth 사용자인 경우 비밀번호 재설정 불가
     if (userData?.isOAuth) {
       throw new Error(
-        "소셜 로그인으로 가입된 계정입니다. 소셜 로그인을 이용해주세요.",
+        "간편 로그인으로 가입된 계정입니다. 소셜 로그인을 이용해주세요.",
```

Comment:
뒷 말도 통일성 있게 간편 로그인으로 바꾸는 거 어때요?

```suggestion
        "간편 로그인으로 가입된 계정입니다. 간편 로그인을 이용해주세요.",
```

Reply:
오잉??? 바꿨습니다 알려주셔서 감사해요!!!